### PR TITLE
strconv.Atoi used for converting string to integer

### PR DESCRIPTION
**Pull request by vthiruveedula**
to integer value

### CODE REVIEW

**Comment by thakurajayL** [comment]
Will this changes have any impact on existing setups ?

File: simapp.go [hunk 1]
```diff
@@ -726,7 +726,7 @@ func UpdateConfig(f string) error {
 					var msg configMessage
 					msg.msgPtr = reqMsgBody
 					msg.msgType = subscriber
-					msg.name = strconv.FormatUint(k, 10)
+					msg.name = strconv.FormatUint(uint64(k), 10)
```
Comment (thakurajayL) [inline simapp.go:729]: will this not remove the 0's at the start of IMSI ? Can you check if message sent towards Webconsole has right URL.  msg.name is used to "            httpend = subscriberHttpend + msg.name"

Comment (vthiruveedula) [reply simapp.go:729]: delete might not work.. let me check

Comment (vthiruveedula) [reply simapp.go]: Thanks Ajay. Delete didn't work. Fixed it and pushed changes. Add /delete of subscribers worked with latest

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**